mw-webapp/src/component/header/Header.module.scss

```diff
@@ -2,14 +2,15 @@
 
 .header {
   display: flex;
-  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
   width: 100%;
-  background: $header-background;
+  padding: 0 1rem;
```

Comment:
why rem?

Reply:
At first I thought of using a percentage, but for some reason I chose rem))
it's all about adaptability